code/modules/mob/living/simple_animal/revenant/revenant_spawn_event.dm

```diff
@@ -1,3 +1,5 @@
+#define REVENANT_SPAWN_THRESHOLD 20
```

Comment:
The dead mob list includes ghosts and dead mobs on the station z-level. By the time this event occurs, there are usually enough dead mobs to make this happen.